SWLOR.Game.Server/Feature/SpawnDefinition/HutlarResourceSpawnDefinition.cs

```diff
@@ -101,7 +102,16 @@ private void FrozenCave()
                 .WithFrequency(40)
                 .AddSpawn(ObjectType.Placeable, "fiberp_bush_4")
                 .WithFrequency(10);
+        }
+        private void QionHiveResources()
+        {
+            _builder.Create("HutlarDungeonLootZiggurat")
```

Comment:
Please follow convention by capitalizing all loot table names/tags. HutlarDungeonLootZiggurat -> HUTLAR_DUNGEON_LOOT_ZIGGURAT as an example. Update any references on creatures to match.

Reply:
Will fix.